daemon/container_operations_unix.go

```diff
@@ -529,6 +529,29 @@ func (daemon *Daemon) updateNetwork(container *container.Container) error {
 	return nil
 }
 
+// UpdateNetworkSettings update the network settings
+func (daemon *Daemon) UpdateNetworkSettings(container *container.Container) error {
```

Comment:
There is already a method `updateNetworkSettings`, so I use `UpdateNetworkSettings` at the very start. what about `updateContainerNetworkSettings`?